rnd/docker-compose.yml

```diff
@@ -93,18 +119,53 @@ services:
     depends_on:
       - postgres
       - redis
+      - migrate
     environment:
       - DATABASE_URL=postgresql://agpt_user:pass123@postgres:5432/agpt_local
       - REDIS_HOST=redis
       - REDIS_PORT=6379
       - REDIS_PASSWORD=password
       - AUTH_ENABLED=false
       - PYRO_HOST=0.0.0.0
+      - PORT=8001
     ports:
       - "8001:8001"
     networks:
       - app-network
 
+  market:
+    build:
+      context: ../
+      dockerfile: rnd/market/Dockerfile
+    depends_on:
+      - postgres
+      - migrate
+    environment:
+      - DATABASE_URL=postgresql://agpt_user:pass123@postgres:5432/agpt_local
```

Comment:
Shouldn't the market have its own separate database?

Reply:
We are going to merge it into the rest server, its pointless having it separate now.